pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -51,7 +54,20 @@ func Convert_v1beta1_KubeSchedulerConfiguration_To_config_KubeSchedulerConfigura
 		return err
 	}
 	out.AlgorithmSource.Provider = pointer.StringPtr(v1beta1.SchedulerDefaultProviderName)
-	return convertToInternalPluginConfigArgs(out)
+	err := convertToInternalPluginConfigArgs(out)
+	if err != nil {
+		return err
+	}
+	registry := frameworkplugins.NewInTreeRegistry()
```

Comment:
It cannot import frameworkplugins for import cycle issue, any suggestions to handle v1beta1 compatibility？@alculquicondor

Reply:
Yeah, we don't want to do that, even if there was no import cycle. I haven't read the PR yet, but what do you want to achieve?

The point of having the qualified names is that we don't need to check that names exist. The domain should match with the api group  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-scheduler/config/v1beta1/register.go#L25

Then we can load that api group into the scheme https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/apis/config/v1beta1/conversion.go#L39
So maybe we can have a registering function.